language_selection.md

```diff
@@ -0,0 +1,86 @@
+# Language selection
+
+Many factors influence how a 18F engineers should make technology selections
+for their projects. Here, we discuss recommendations on how to select the
+_language_ used in your projects. This document doesn't offer hard-and-fast
+rules, focusing instead on the aspects which should be balanced against each
+other. Also, we explicitly note that language selection is ultimately the
+decision (and responsibility) of the project team. The guidance below is meant
+to help your team make this decision.
+
+This is a living document. After looking at existing tickets, start an issue
+or pull request in GitHub to suggest changes to these standards.
+
+## Our strongest languages
+
+18F has historically worked in three primary languages: JavaScript, Python,
+and Ruby. While we certainly have a healthy smattering of other odds and ends,
+these three are our primary strength as indicated by data from past projects,
+engineer skill sets, and billable hours. These languages are generally a safe
+bet as we know we can support them and be productive with them.
+
+Looking at this same data and applying preferences from the guiding factors
+below, we also see a second tier of familiarity around Go, Java, and PHP.
+These languages may be good fits depending on the project at hand. In
+particular, domain preference and client familiarity may be major reasons to
+select one of these languages for your project. We feel relatively comfortable
+supporting these languages, but if all other factors are equal, we encourage
+selecting from our first tier.
```

Comment:
I like the way these paragraphs are laid out -- using two headers might make it easier for someone to parse this via scanning. Maybe h3 called "Tier 1 languages" and another called "Tier 2 languages" with same content?